src/pages/FilesPage/components/FolderItem/FolderItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Typography from '@mui/material/Typography';
+import {
+	FolderItemCard,
+	FolderItemCheckbox,
+	FolderItemContainer,
+	FolderItemIcon,
+} from './FolderItem.styles';
+import { FolderResponse } from 'clients/CoreService';
+import { renderLongName } from '../renderers/renderLongName';
+import { getBasename } from 'shared/util';
+
+export interface FolderItemProps {
+	folder: FolderResponse;
+	checked?: boolean;
+	isCheckedFile: boolean;
+	setChecked: React.Dispatch<React.SetStateAction<string[]>>;
+	setLocation: React.Dispatch<React.SetStateAction<string>>;
+}
+
+export function FolderItem({
+	folder,
+	checked,
+	isCheckedFile,
+	setChecked,
+	setLocation,
+}: FolderItemProps) {
+	const handleClick = () => {
+		if (!folder.path || isCheckedFile) return;
+		if (checked) {
+			setChecked(ids => {
+				return ids.filter(basename => basename !== folder.path);
+			});
+		} else {
+			setChecked(checked => [...checked, folder.path]);
+		}
+	};
+
+	const folderName: string = getBasename(folder.path);
```

Comment:
```suggestion
	const folderName: string = basename(folder.path);
```
Use "path" lib instead